report/assets/styles.css

```diff
@@ -1496,14 +1495,20 @@
 }
 
 .lh-table {
-  border-collapse: collapse;
+  position: relative;
+  border-collapse: separate;
+  border-spacing: 0;
   /* Can't assign padding to table, so shorten the width instead. */
   width: calc(100% - var(--audit-description-padding-left) - var(--stackpack-padding-horizontal));
   border: 1px solid var(--report-border-color-secondary);
-
 }
 
 .lh-table thead th {
+  position: sticky;
+  top: calc(var(--topbar-plus-sticky-header) + 1em);
+  z-index: 1;
+  background-color: var(--report-background-color);
+  border-bottom: 1px solid var(--highlighter-background-color);
```

Comment:
Personally I think this color is too loud and `--report-border-color-secondary` works better.